ecl/hql/hqllex.l

```diff
@@ -871,6 +871,7 @@ RECORD              { RETURNHARD(RECORD); }
 RECORDOF            { RETURNSYM(RECORDOF); }
 RECOVERY            { RETURNSYM(RECOVERY); }
 REGEXFIND           { RETURNSYM(REGEXFIND); }
+REGEXFINDSET        { RETURNSYM(REGEXFINDSET); }
```

Comment:
You also need a matching entry in hqlgram2.cpp getTokenText(). 
eclcc -internal
should catch any issues like this.